app/controllers/admin/agents/rdvs_controller.rb

```diff
@@ -0,0 +1,23 @@
+class Admin::Agents::RdvsController < ApplicationController
+  include Admin::AuthenticatedControllerConcern
+  respond_to :json
+
+  def index
+    agent = Agent.find(params[:agent_id])
+    @organisation = Organisation.find(params[:organisation_id])
+    @rdvs = custom_policy.with_agent(agent)
```

Comment:
```suggestion
    @rdvs = custom_policy.where(agent: agent)
```
?

J’aime bien `.merge` sinon, mais c’est un peu overkill ici:
```suggestion
    @rdvs = custom_policy.merge(Agent.rdvs)
```

Reply:
Je ne connaissais pas celui là :)

Reply:
C’est vachement bien, ça permet de grouper plusieurs fragments de queries construites indépendamment. Par contre, je trouve le nom mal choisi, ça ne veut pas dire “faire l’union des résultats”.

Reply:
Ah oui :). Je vais garder le `where(agent: agent)` pour le moment.